TarSCM/scm/git.py

```diff
@@ -251,7 +251,7 @@ def prepare_working_copy(self):
         # between multiple services
         org_clone_dir  = self.clone_dir
         self.clone_dir = self.repodir
-        command = ['git', 'clone', self.repocachedir, self.url, self.clone_dir]
+        command = ['git', 'clone', '--no-checkout']
```

Comment:
What workflow is that exactly? Isn't this clone happening in an ephemeral tmpdir? If there's another way this is used please let me know and I'm happy to fix or remove